code/modules/mob/living/simple_animal/bot/griefsky.dm

```diff
@@ -185,29 +185,44 @@
 /mob/living/simple_animal/bot/secbot/griefsky/explode()
 	walk_to(src,0)
 	visible_message("<span class='boldannounceic'>[src] lets out a huge cough as it blows apart!</span>")
-	var/turf/Tsec = get_turf(src)
-	new /obj/item/assembly/prox_sensor(Tsec)
-	var/obj/item/secbot_assembly/Sa = new /obj/item/secbot_assembly(Tsec)
+	var/turf/explode_turf = get_turf(src)
+	new /obj/item/assembly/prox_sensor(explode_turf)
+	var/obj/item/secbot_assembly/Sa = new /obj/item/secbot_assembly(explode_turf)
```

Comment:
while you're changing vars, can you change this variable to something like `assembly`?

Reply:
I forgot griefskies were made with helmets as well. I'll change it to `pierced_helmet` like I did for the ED209 and securitron (I feel like naming it assembly makes the ED209's code harder to read since it also has the reinforced cyborg chassis assembly thing)